guides/doc-Configuring_Load_Balancer/topics/Configuring_Capsule_Server_with_Default_SSL_Certificates_for_Load_Balancing_with_Puppet.adoc

```diff
@@ -68,12 +68,20 @@ root@_{smart-proxy-context}_ca.example.com_:__{smart-proxy-context}_ca.example.c
 --enable-foreman-proxy-plugin-remote-execution-ssh
 ----
 
-. On {SmartProxyServer}, generate Puppet certificates for all other {SmartProxyServer}s that you configure for load balancing, except this first system where you configure Puppet certificates signing:
+. On {SmartProxyServer}, generate Puppet certificates for all other {SmartProxyServer}s that you configure for load balancing, except the first system where you configure Puppet certificates signing:
+.. First, stop the Puppet agent:
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# puppet cert generate _{smartproxy-example-com}_ \
---dns_alt_names=_loadbalancer.example.com_
+# puppet resource service puppetserver ensure=stopped
+----
++
+.. Generate certificate for the {SmartProxyServer}:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# puppetserver ca generate --certname _{smartproxy-example-com}_ \
```

Comment:
Why is this needed? You're already passing all the options in the installer command. This doesn't make sense to me unless there's a bug.

Reply:
Apologies, I was a bit too quick. Looks like this may indeed be needed. I'm not sure if the other Puppetservers are configured to use the bits that use `--ca-client`.

I would also add a note that `--ca-client` should only be used for the load balanced servers. If people use it to create certificates for regular nodes, it creates a big security issue.

Reply:
No worries. Added a warning admonition/note. thanks.